src/main/java/mate/academy/controller/CategoryController.java

```diff
@@ -43,8 +43,8 @@ public CategoryDto createCategory(@RequestBody @Valid CreateCategoryRequestDto r
             description = "Returns a list of all available categories")
     @PreAuthorize("hasRole('ROLE_USER')")
     @GetMapping
-    public List<CategoryDto> findAllCategories() {
-        return categoryService.findAll();
+    public List<CategoryDto> findAllCategories(Pageable pageable) {
```

Comment:
please add default parameters for your pageable

Reply:
For pagination, the Pageable object contains the attributes: _page_ and _size_. If you do not pass these fields, the default values are _page = 0_ and _size = 20_.